efsity/src/main/java/org/smartregister/command/TranslateCommand.java

```diff
@@ -93,6 +90,9 @@ public void run() {
         // Check if the input path is a directory or a JSON file
         if (Files.isDirectory(inputFilePath)) {
           if (Objects.equals(extractionType, "configs") || inputFilePath.endsWith("configs")) {
+            // handle case where extractionType has not been given and inputFilePath ends with
+            // configs
+            if (Objects.equals(extractionType, null)) extractionType = "configs";
```

Comment:
This is okay since one is null. This approach is preferred when both are objects